docs/generate-time-series/go.rst

```diff
@@ -339,20 +357,30 @@ Then in the script's ``main`` function, create an infinite loop that gets the
                     fmt.Println("INSERT OK")
                 }
             }
-            fmt.Println("Sleeping for 5 seconds...")
-            time.Tick(time.Second * 5)
+            fmt.Println("Sleeping for 10 seconds...")
+            time.Tick(time.Second * 10)
 	    }
     }
 
+.. SEEALSO::
```

Comment:
@infoverload moved the link to the source so it appears directly after the code block so the reader knows right away they can copy from there instead. also: updated the code to match the new formatting